src/Microsoft.Graph.Core/Requests/IBaseRequest.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Microsoft.Graph
```

Comment:
same thing, probably not needed anymore.

Reply:
To avoid making the PR too large and easier to review(and still have it compile) My intention is to refactor IBaseRequest after closing #334. 
If it's okay you will see this is done there.